src/Test/Utilities/Desktop/CommonDiagnosticAnalyzers.cs

```diff
@@ -480,120 +480,5 @@ private void OnCompilation(CompilationAnalysisContext context)
                 }
             }
         }
-
-        /// <summary>
-        /// This analyzer is intended to be used only when concurrent execution is enabled for analyzers.
-        /// This analyzer will deadlock if the driver runs analyzers on a single thread OR takes a lock around callbacks into this analyzer to prevent concurrent analyzer execution
-        /// Former indicates a bug in the test using this analyzer and the latter indicates a bug in the analyzer driver.
-        /// </summary>
-        [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
-        public class ConcurrentAnalyzer : DiagnosticAnalyzer
```

Comment:
It required the semaphoreslim extensions reverted with this PR.  I can put those back and see if it compiles.